package.json

```diff
@@ -27,6 +27,11 @@
         "command": "angular.openLogFile",
         "title": "Open Angular Server log",
         "category": "Angular"
+      },
+      {
+        "command": "angular.getTemplateTcb",
```

Comment:
I wonder if it would be good to provide this command in the context menu of html files: https://stackoverflow.com/a/41400631

Reply:
neat idea! done.